.github/workflows/build.yml

```diff
@@ -55,57 +62,146 @@ jobs:
       - name: Build docker image
         run: | 
           SAMPLE_LIST=(hello-kubernetes distributed-calculator pub-sub bindings observability secretstore)
-
           for sample in "${SAMPLE_LIST[@]}"; do
             echo "Building image for ${sample}..."
             pushd ${sample}
             make build
             popd
           done
-      - name: docker login
-        run: |
-          docker login -u ${{ secrets.DOCKER_REGISTRY_ID }} -p ${{ secrets.DOCKER_REGISTRY_PASS }}
-      - name: publish image to dockerhub
+     - name: GitHub container registry login
+        if: matrix.target_os != 'darwin'
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push docker images to GitHub container registry
+        if: matrix.target_os != 'darwin'
```

Comment:
Tanvi, this change does not make sense because it is cloning all the workflow from dapr/dapr. We need to change the current quickstart workflow to do a push to GHCR in addition to Docker Hub. The change is probably much smaller.

Reply:
@artursouza As discussed earlier, In case of Docker we publish quickstarts images to a different repository than Dapr-
https://github.com/dapr/quickstarts/blob/870ed621b843708cab9a78f7ec4cd122575957c2/docker.mk#L8
With this PR, quickstarts images also will be published at the same place as Dapr components.

Reply:
Yes, they will. We just add them under `/samples` folder. I talked to @yaron2 about that already. This PR, on the other hand, does not work at all for quickstarts.